docs/import/import_data.js

```diff
@@ -68,17 +62,31 @@ function countDamageAndBuildings(feature, buildings) {
           // non-number values like "-" :(
           .set(incomeTag, feature.get(incomeTag))
           .set(sviTag, feature.get(sviTag))
-          .set(buildingCountTag, totalBuildings)
-          .set(damageTag, ratioBuildingsDamaged));
+          .set(buildingCountTag, totalBuildings);
+  if (damage) {
+    const damagedBuildings =
+        ee.FeatureCollection(damage).filterBounds(geometry).size();
+    properties = properties.set(
+        damageTag,
+        // If no buildings, this is probably spurious. Don't give any damage.
+        ee.Algorithms.If(
+            totalBuildings, ee.Number(damagedBuildings).divide(totalBuildings),
```

Comment:
Yup. But I think a no-building area is likely to be unimportant to GD anyway. If CrowdAI has a false positive in the middle of a forest, I think we want to filter it out.

Reply:
well, the "0" branch will be triggered if totalBuildings evaluates to false. EarthEngine says in their docs:

> The condition that determines which result is returned. If this is not a boolean, it is interpreted as a boolean by the following rules:
> - Numbers that are equal to 0 or a NaN are false.
> - Empty strings, lists and dictionaries are false.
> - Null is false.
> - Everything else is true.

Since totalBuildings is either 0, or the value of a histogram, it is a number. So unless we somehow got NaN from the histogram, totalBuildings being false indicates we found no buildings in that block group. Is that what you were asking? Sorry if I misunderstood.